.travis.yml

```diff
@@ -7,6 +7,19 @@ jobs:
   allow_failures:
     - rust: nightly
   fast_finish: true
+  include:
+    - name: "Rust: stable - Examples"
+      language: rust
+      rust: stable
+      install:
+        - (cd examples/truffle; npm ci)
+      script:
+        - (cd examples/truffle; npm run build)
+        - docker run -d -p 9545:9545 trufflesuite/ganache-cli:v6.7.0 -p 9545
```

Comment:
double p parameter?

Reply:
One to docker one to ganache

Reply:
Is `-p 8545:9545` allowed for mapping ports in docker?